tests/src/queue/src/main.sw

```diff
@@ -0,0 +1,45 @@
+script;
+
+use queue::Queue;
+use std::assert::assert;
```

Comment:
```suggestion
```

Assert is in the prelude so we can remove it

Reply:
Removed Rust tests